src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -231,6 +243,12 @@ public async Task OnEndProcessTask(string taskId, Instance instance)
                 }
             }
 
+            if (_appSettings != null && _appSettings.EnableEFormidling && _appMetadata.EFormidling.SendAfterTaskId == taskId)
+            {
+                string token = JwtTokenUtil.GetTokenFromContext(_httpContextAccessor.HttpContext, _appSettings.RuntimeCookieName);
```

Comment:
Feels like a nasty hack, does anyone know a better resolution? 
When retrieving the binary data to send to eFormidling we need to get the token of the user from the HttpContext, however, between this point and now, we have accessed the app service which is transient, resulting in the HttpContext to be wiped clean by the time we return to the point where we are ready to retrieve the binary data. 
@SandGrainOne @TheTechArch